samples/inference.md

```diff
@@ -6,337 +6,353 @@ We will show:
    * REST and gRPC using seldon CLI, curl and grpcurl
  * Pipeline inference
    * REST and gRPC using seldon CLI, curl and grpcurl
- 
-
 
 
 ```python
 %env INFER_ENDPOINT=0.0.0.0:9000
```

Comment:
:question: Do we think this should change?  This magic is essentially doing `export INFER_ENDPOINT=...`, so we could detect this and format it as a bash block.

If so, I'd prefer to add a follow-up PR, as this one is already fairly large and complex.